com.unity.render-pipelines.universal/ShaderLibrary/Debug/DebugViewEnums.cs

```diff
@@ -3,110 +3,232 @@
 namespace UnityEngine.Rendering.Universal
 {
     #region Material Settings
+
+    /// <summary>
+    /// Debug material modes.
+    /// </summary>
     [GenerateHLSL]
     public enum DebugMaterialMode
```

Comment:
If feels weird that the Debug API is internal but we expose this as public. Is there any usage for user for this to be public without debug API?

Reply:
I don't think there's ways for users to actually use these. These are also public in [the equivalent HDRP file](https://github.com/Unity-Technologies/Graphics/blob/master/com.unity.render-pipelines.high-definition/Runtime/Debug/LightingDebug.cs), so I thought I'd document them rather than make them internal. I guess it would be useful to generate documentation from this file and link to it from the feature docs, but I can also mark them internal if preferred.